src/ggrc/assets/javascripts/components/assessment_attributes.js

```diff
@@ -0,0 +1,103 @@
+/*!
+    Copyright (C) 2016 Google Inc., authors, and contributors <see AUTHORS file>
+    Licensed under http://www.apache.org/licenses/LICENSE-2.0 <see LICENSE file>
+    Created By: ivan@reciprocitylabs.com
+    Maintained By: ivan@reciprocitylabs.com
+*/
+
+(function (can, $) {
+  can.Component.extend({
+    tag: 'assessment-template-attributes',
+    template: '<content></content>',
```

Comment:
Isn't this element [deprecated](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/content)? Please double check.